pkg/compliance/report.go

```diff
@@ -21,8 +21,10 @@ type Report struct {
 	Aggregated bool
 	// Evaluator defines the eval engine that was used to generate this report
 	Evaluator string
-	// Error of th check evaluation
+	// Error of the check evaluation
 	Error error
+	// UserProvidedError indicates if the error was provided by the user rule
+	UserProvidedError bool
```

Comment:
Will it be in the log sent to the backend ?

Reply:
@lebauce what is sent to the backend is defined in https://github.com/DataDog/datadog-agent/blob/51c61c5da6f6e3989c326ff8fcbb5131b8bc1529/pkg/compliance/event/event.go#L23

Reply:
And constructed in https://github.com/DataDog/datadog-agent/blob/51c61c5da6f6e3989c326ff8fcbb5131b8bc1529/pkg/compliance/checks/check.go#L141